pkg/dashboard/handlers/helmHandlers.go

```diff
@@ -351,22 +352,17 @@ func (h *HelmHandler) Install(c *gin.Context) {
 }
 
 func (h *HelmHandler) Tests(c *gin.Context) {
-	qp, err := utils.GetQueryProps(c, false)
-	if err != nil {
-		_ = c.AbortWithError(http.StatusBadRequest, err)
-		return
+	rel, _ := h.getRelease(c)
+	if rel == nil {
+		return // error state is set inside
 	}
 
-	panic("Needs implementing")
-	out, _ := "", qp
-
-	//out, err := h.Data.RunTests(qp.Namespace, qp.Name)
+	out, err := rel.Tests()
```

Comment:
I'd rename it into `RunTests`, 'cause right now it sounds like "get list of tests"